user-docs/documentation/docs/how-to-guides/1-use-serverless-contracts/1-api-gateway.md

```diff
@@ -316,11 +316,40 @@ The `getLambdaHandler` is only a pass-through adding typing capabilities, withou
 This method is not recommended, use `getHandler` when possible.
 :::
 
-In order to safely use `getLambdaHandler` combine it with `applyHttpMiddlewares`
+In order to safely use `getLambdaHandler`, you need to define a custom middleware than you can use to wrap your handler.
+Here is an implementation example:
+
+```ts
```

Comment:
You should add the `title="path/to/file.ts` to make it easier to understand where this file is located/what it is